src/KrakenBattle.cpp

```diff
@@ -119,13 +119,16 @@ namespace KrakenBattle {
       auto sendingResult = SendChunks(kraken, uuid, sendState, chunk, error);
       bool result = (Kraken::Battling::CONTINUE == sendingResult);
       LOG_IF(WARNING, (!result)) << "Failed to send 'SendTidalWave'" << ", uuid: " << uuid
-                                 << ", type: " << KrakenBattle::EnumToString(sendState);
+                                 << ", type: " << KrakenBattle::EnumToString(sendState)
```

Comment:
I'm sure Uncle Bob would approve if you changed "type" to "sendState", since that's what you are printing.